docs/src/pages/api-reference/expo.mdx

```diff
@@ -0,0 +1,171 @@
+import { Callout } from "nextra-theme-docs";
+
+## `generateReactNativeHelpers`
+
+The `generateReactNativeHelpers` function is used to generate the
+useImageUploader and useDocumentUploader hooks you use to interact with
+UploadThing in your app.
+
+```tsx copy filename="utils/uploadthing.tsx"
+import { generateReactNativeHelpers } from "@uploadthing/expo";
+
+import type { UploadRouter } from "~/app/api/uploadthing+api";
+
+export const { useImageUploader, useDocumentUploader } =
+  generateReactNativeHelpers<UploadRouter>({
+    /**
+     * Your server url.
+     * @default process.env.EXPO_PUBLIC_SERVER_URL
+     * @remarks In dev we will also try to use Expo.debuggerHost
+     */
+    url: "https://my-server.com",
```

Comment:
Is there any way to "hook" into this variable @EvanBacon?

![CleanShot 2024-04-18 at 12 06 07](https://github.com/pingdotgg/uploadthing/assets/51714798/4d175d27-8ecc-4080-ac4e-dd0aee63308f)